File: frontend/src/app/service/rest-response.ts
import {ResponseModel} from "./response-model";

export class RestResponse{
  status: number;
  info: string;
  data: Array<ResponseModel>;
  answer: string;

  constructor(status: number, info: string, data: Array<ResponseModel>, answer: string) {
    this.status = status;
    this.info = info;
    this.data = data;
    this.answer = answer;
  }
}

File: Backend/src/main/java/diploma/lukaric/backend/RestResponse.java
package diploma.lukaric.backend;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.ArrayList;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class RestResponse {
    private int status;
    private String info;
    private ArrayList<ResponseModel> data;

    private String answer;
}
